index.html

```diff
@@ -35,5 +35,149 @@ <h1>Norbert Siko</h1>
     </section>
 
     <div class="img-background"></div>
+
+    <section class="work">
+      <h1 class="projects">Projects</h1>
```

Comment:
> - [ ] I suggest you use an `h2 `tag instead of an `h1`. By using appropriate heading levels, you can help search engines understand the main topics and subtopics of a page, which can improve its ranking in search results.👍